doc/api/api_changes/2017-30-08_AFV_deprecation_of_axes_collision.rst

```diff
@@ -0,0 +1,15 @@
+Deprecation of axes collision
+-----------------------------
+
+The following is related to the PR #9037 (and incidentally to #9034 too).
```

Comment:
can you tag these as 
```

:issue:`XXXX`

```

so they link right?

Reply:
I would also move them to the *bottom* of the message.  The relevant issues are actually #7377 (where most of the rationale is discussed) and #9024; #9034 and #9037 are just technicalities.

Reply:
Rephrased the mention to use the issue # suggestd by @anntzer and moved it at the end of the entry. Plus now using the `:issue:` tag.